exodus_gw/migrations/versions/5bd0b38df850_.py

```diff
@@ -0,0 +1,60 @@
+"""empty message
+
+Revision ID: 5bd0b38df850
+Revises: 8b70b7e9c7fc
+Create Date: 2022-08-19 13:57:38.431351
+
+"""
+import os
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "5bd0b38df850"
+down_revision = "8b70b7e9c7fc"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("DELETE FROM tasks")
+    op.execute("DELETE FROM publishes")
+
+    # recreate='always' to avoid "Cannot add a NOT NULL column with default value NULL"
+    # on sqlite < 3.32.0
+    recreate = (
+        "always"
+        if "sqlite" in os.environ.get("EXODUS_GW_DB_URL", "")
+        else "auto"
+    )
+
+    with op.batch_alter_table("tasks", recreate=recreate) as batch_op:
+        batch_op.alter_column(
+            "updated", existing_type=sa.DateTime(), nullable=True
```

Comment:
This doesn't work because the correct argument name is `type_`, not `existing_type`.  See the docs at https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.alter_column .  If you change it like this it seems to work:

```suggestion
            "updated", type_=sa.DateTime(), nullable=True
```

I understand you would have copied this from an existing migration.  I believe all the current migrations using `existing_type` are also wrong.